explainers/content-addressable-bundles.md

```diff
@@ -549,6 +561,26 @@ and `https://example.com/daily-host-contents.00c0.wbn` because the browser
 probably can load `cold-contents.js.00a1.wbn`, whose hash didn't change, from
 its cache.
 
+### I would like to separate code into bundles for various reasons, but I'm afraid this causes too many round-trips if bundles are nested and discovered one bye one. Is there any good way to avoid it?
+
+A possible workaround as a user-land solution would be to declare indirect
+dependent bundles as well as direct dependent bundles.
+
+In the example case, the top page may declare:
+
+<!-- prettier-ignore -->
+```html
+<!-- This is a mandarory -->
+<link rel=webbundle href=https://example.com/app/index.js.abcd.wbn resources="index.js" >
+<!-- The followings are optional, as a hint for a browwer to prefetch -->
```

Comment:
s/browwer/browser/

Reply:
Ah, sorry for that. Thanks!
Let me fix typo, and merge with a fix.